### PR TITLE
Add support for line breaks in AMP exporter

### PR DESCRIPTION
**Pull request by towanda**

### CODE REVIEW

File: lib/article_json/export/amp/elements/base.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ def initialize(element)
           # Export a HTML node out of the given element
           # Dynamically looks up the right export-element-class, instantiates it
           # and then calls the #build method.
-          # @return [Nokogiri::HTML::Node]
+          # @return [Nokogiri::HTML::NodeSet]
```
Comment (dsager) [inline lib/article_json/export/amp/elements/base.rb]: should be `XML::NodeSet`, i guess

File: lib/article_json/export/amp/elements/heading.rb [hunk 2]
```diff
@@ -3,6 +3,8 @@ module Export
     module AMP
       module Elements
         class Heading < Base
+          # Generate the heading element with the right text
+          # @return [Nokogiri::XML::Element]
```
Comment (dsager) [inline lib/article_json/export/amp/elements/heading.rb]: won't this also be a NodeSet?

**Comment by miguelbernadi** [comment]
+1

File: lib/article_json/export/amp/elements/shared/caption.rb [hunk 3]
```diff
@@ -6,9 +6,9 @@ module Shared
           module Caption
             # Generate the caption node
             # @param [String] tag_name
-            # @return [Nokogiri::HTML::Node]
+            # @return [Nokogiri::HTML::NodeSet]
```
Comment (dsager) [inline lib/article_json/export/amp/elements/shared/caption.rb]: HTML -> XML

File: lib/article_json/export/amp/elements/text.rb [hunk 4]
```diff
@@ -13,29 +15,38 @@ def export
 
           private
 
-          # @return [Nokogiri::HTML::Node]
+          # @return [Nokogiri::HTML::NodeSet]
```
Comment (dsager) [inline lib/article_json/export/amp/elements/text.rb]: HTML -> XML

File: lib/article_json/export/amp/elements/text.rb [hunk 5]
```diff
@@ -3,6 +3,8 @@ module Export
     module AMP
       module Elements
         class Text < Base
+          # Generate a Nokogiri node or simple text node, depending on the text
+          # options
           # @return [Nokogiri::HTML::Node]
```
Comment (dsager) [inline lib/article_json/export/amp/elements/text.rb]: => `Nokogiri::XML::NodeSet`

**Comment by towanda** [comment]
@dsager fixed comments, thanks!

**Comment by christofdamian** [comment]
+1

**Comment by dsager** [comment]
+1

**Pull request merged**